addons/binding/org.openhab.binding.seneye/src/main/java/org/openhab/binding/seneye/internal/SeneyeService.java

```diff
@@ -29,39 +27,43 @@
 /**
  * The {@link SeneyeService} handles the connection to the Seneye API
  *
- * @author Niko Tanghe
+ * @author Niko Tanghe - Initial contribution
  */
 
 public class SeneyeService {
 
     private static Logger logger = LoggerFactory.getLogger(SeneyeService.class);
 
-    private long refreshInterval;
     private int retry;
 
-    private String aquariumname;
-    private String userName;
-    private String password;
+    private SeneyeConfigurationParameters config;
+
     private String seneyeId;
 
     private boolean isInitialized;
 
     private final Gson gson;
 
-    public SeneyeService() {
-        this("", "", "", 120000);
-    }
+    private static HttpClient httpClient = new HttpClient(new SslContextFactory());
+
+    public SeneyeService(SeneyeConfigurationParameters config) {
+        this.config = config;
 
-    public SeneyeService(String aquariumname, String userName, String password, long refreshInterval) {
-        this.aquariumname = aquariumname;
-        this.userName = userName;
-        this.password = password;
-        this.refreshInterval = refreshInterval;
         this.retry = 1;
 
         this.gson = new Gson();
 
         this.isInitialized = false;
+
+        if (!httpClient.isStarted()) {
+            try {
+                httpClient.setFollowRedirects(false);
+                httpClient.start();
+            } catch (Exception e) {
+                logger.warn("Cannot start HttpClient!", e);
```

Comment:
Please change this to debug log level.

Reply:
I think this is an error, if httpclient is not configured, it will throw an exception on the first use

Reply:
If it is really an error, then why not logging it on `error` level? ;-)

Reply:
ok, thing is set OFFLINE now if HttpClient initialization fails for some reason.